salt/orch/update.sls

```diff
@@ -26,39 +33,26 @@ generate-sa-key:
     - sls:
       - kubernetes-common.generate-serviceaccount-key
 
-# Generic Updates
-sync-pillar:
-  salt.runner:
-    - name: saltutil.sync_pillar
```

Comment:
We need this `saltutil.sync_pillar` explicitly with `salt.runner`, that's how the master will load the `_pillars/velum.py` on the master.

Reply:
Oh, I missed it. I will add it in the `update-data` block. Thanks.